### DISCUSSION TITLE
invalid login Expected element type <ErrorResponse> but have <Error>

### DISCUSSION THREAD

**Discussion by kodkurdu** [minio/minio#14077] [Q&A] [answered]
Hi all,
I have installed MinIO in Distributed Mode on 3 host which has oracle linux 8.5 and nginx load blancer is frontend.  

4 oracle linux 8.5 on rhv 4.4.8.5-0.4.el8ev
1 LB nginx 
3 Minio host

I follewed this document is https://docs.min.io/minio/baremetal/installation/deploy-minio-distributed.html, I did not use ssl. 

My minio config is

MINIO_VOLUMES="http://minio{1...3}.test.local/mnt/disk{1...4}/minio"
MINIO_OPTS="--console-address :9001"
MINIO_ROOT_USER=minioadmin
MINIO_ROOT_PASSWORD=minioadmin
MINIO_SERVER_URL="http://minio.test.local"

Installation has been completly successfull, I have riched web ui but neither I could not login http://minio.test.local nor http://minio1.test.local:9001 and another hosts.  I accessed minio client to minio cluster and added alias (mc alias s ev1 http://minio1.test.local:9000 minioadmin minioadmin) after that listed clusters host info. I don't understand why I cannot login ui.
Thanks for answers.
  
 ![loginFailed](https://user-images.githubusercontent.com/36971305/148845023-59e03dec-6a67-4c0e-9c3d-2023025fc1e6.PNG)

![mc-status](https://user-images.githubusercontent.com/36971305/148847638-f3f29934-f149-41aa-a5e9-2b1042aa2c8c.PNG)

**Comment by harshavardhana** [comment, marked as answer]
> 4 oracle linux 8.5 on rhv 4.4.8.5-0.4.el8ev 1 LB nginx 3 Minio host
> 
> I follewed this document is https://docs.min.io/minio/baremetal/installation/deploy-minio-distributed.html, I did not use ssl.
> 
> My minio config is
> 
> MINIO_VOLUMES="[http://minio{1...3}.test.local/mnt/disk{1...4}/minio](http://minio%7B1...3%7D.test.local/mnt/disk%7B1...4%7D/minio)" MINIO_OPTS="--console-address :9001" MINIO_ROOT_USER=minioadmin MINIO_ROOT_PASSWORD=minioadmin MINIO_SERVER_URL="http://minio.test.local"

It cannot login because you didn't fix your nginx configuration properly. You are pointing to console URL port instead of pointing to MinIO port for login.

**Comment by harshavardhana** [comment]
Make sure to provide `mc admin trace -a` output and attach it here.

**Reply by kodkurdu** [reply]
I added it

**Comment by harshavardhana** [comment]
You need to save the file and attach the output here @kodkurdu

**Reply by kodkurdu** [reply]
Sorry, my fault. I added it
[cikti.txt](https://github.com/minio/minio/files/7845275/cikti.txt)
I installed with minio-20220108031154.0.0.x86_64.rpm

**Reply by kodkurdu** [reply]
I removed MINIO_SERVER_URL="http://minio.test.local" from minio config file and I have loged in minio ui on minio host. Can we say this problem occured because of nginx?

**Reply by harshavardhana** [reply]
> I removed MINIO_SERVER_URL="http://minio.test.local" from minio config file and I have loged in minio ui on minio host. Can we say this problem occured because of nginx?

yes you are adding incorrect MINIO_SERVER_URL values @kodkurdu

**Reply by kodkurdu** [reply]
I have learned hard way :). Does MINIO_SERVER_URL have to be a value? If yes, what you suggest to me?

**Reply by harshavardhana** [reply]
> I have learned hard way :). Does MINIO_SERVER_URL have to be a value? If yes, what you suggest to me?

You should read why it is needed https://github.com/minio/minio#test-minio-connectivity its not always needed.

**Comment by answerquest** [comment]
Fixed at my end by removing MINIO_SERVER_URL and keeping just MINIO_BROWSER_REDIRECT_URL . I've setup on an Apache server though, not Nginx, so this isn't directly applicable to OP's question. But this issue comes up in common search queries so may be helpful to some.
